emitter/emitter.go

```diff
@@ -124,6 +136,62 @@ func EmitChanges(emission Emission) {
 	}
 }
 
+// EmitChanges sends a json string payload of cluster changes to a vRO remote endpoint
```

Comment:
```suggestion
// EmitChangesVRO sends a json string payload of cluster changes to a vRO remote endpoint
```
Small nit for comment.